content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -3,59 +3,77 @@ reviewers:
 - fgrzadkowski
 - jszczepkowski
 - directxman12
-title: Horizontal Pod Autoscaler
+title: Horizontal Pod Autoscaling
 feature:
   title: Horizontal scaling
   description: >
     Scale your application up and down with a simple command, with a UI, or automatically based on CPU usage.
-
 content_type: concept
 weight: 90
 ---
 
 <!-- overview -->
 
-The Horizontal Pod Autoscaler automatically scales the number of Pods
-in a replication controller, deployment, replica set or stateful set based on observed CPU utilization (or, with
-[custom metrics](https://git.k8s.io/community/contributors/design-proposals/instrumentation/custom-metrics-api.md)
-support, on some other application-provided metrics). Note that Horizontal
-Pod Autoscaling does not apply to objects that can't be scaled, for example, DaemonSets.
+In Kubernetes, a _HorizontalPodAutoscaler_ automatically updates a workload resource (such as
+a {{< glossary_tooltip text="Deployment" term_id="deployment" >}} or
+{{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}), with the
+aim of automatically scaling the workload to match demand.
 
-The Horizontal Pod Autoscaler is implemented as a Kubernetes API resource and a controller.
-The resource determines the behavior of the controller.
-The controller periodically adjusts the number of replicas in a replication controller or deployment to match the observed metrics such as average CPU utilisation, average memory utilisation or any other custom metric to the target specified by the user.
+Horizontal scaling means that the response to increased load is to deploy more
+{{< glossary_tooltip text="Pods" term_id="pod" >}}.
+This is different from _vertical_ scaling, which for Kubernetes would mean
+assigning more resources (for example: memory or CPU) to the Pods that are already
+running for the workload.
+
+If the load decreases, and the number of Pods is above the configured minimum,
+the HorizontalPodAutoscaler instructs the workload resource (the Deployment, StatefulSet,
+or other similar resource) to scale back down.
 
+Horizontal pod autoscaling does not apply to objects that can't be scaled (for example:
+a {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}.)
 
+The HorizontalPodAutoscaler is implemented as a Kubernetes API resource and a
+{{< glossary_tooltip text="controller" term_id="controller" >}}.
+The resource determines the behavior of the controller.
+The horizontal pod autoscaling controller, running within the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}, periodically adjusts the
+desired scale of its target (for example, a Deployment) to match observed metrics such as average
+CPU utilisation, average memory utilisation,or any other custom metric you specify.
```

Comment:
Is that the British way to spell "utilization"? :-)

Reply:
I think we should use consistent spelling of the term throughout the document, although I don't mind which one you use. Both seem to be correct.